apps/studio/components/interfaces/DiskManagement/DiskMangement.types.ts

```diff
@@ -0,0 +1,43 @@
+import { AddonVariantId, ProjectAddonVariantMeta } from 'data/subscriptions/types'
+
+export type ComputeInstanceAddonVariantId =
+  | Extract<
```

Comment:
odd these aren't in some other location?

Reply:
🤷‍♂️ if someone finds them, lemme know.